ui/src/views/Quickstart/index.jsx

```diff
@@ -423,87 +446,98 @@ export default class QuickStart extends Component {
+                        />
+                      }
+                      label="Pull request opened"
+                    />
+                    <FormControlLabel
+                      className={classes.formLabelSpacing}
+                      control={
+                        <Checkbox
+                          checked={events.has('pull_request.closed')}
+                          onChange={this.handleEventsSelection}
+                          value="pull_request.closed"
+                        />
+                      }
+                      label="Pull request merged or closed"
+                    />
+                    <FormControlLabel
+                      className={classes.formLabelSpacing}
+                      control={
+                        <Checkbox
+                          checked={events.has('pull_request.synchronize')}
+                          onChange={this.handleEventsSelection}
+                          value="pull_request.synchronize"
+                        />
+                      }
+                      label="New commit made in an opened pull request"
                     />
-                  }
-                  label="Release or tag created"
-                />
+                  </FormGroup>
+                  <FormGroup>
+                    <FormControlLabel
+                      className={classes.formLabelSpacing}
+                      control={
+                        <Checkbox
+                          checked={events.has('pull_request.reopened')}
+                          onChange={this.handleEventsSelection}
+                          value="pull_request.reopened"
+                        />
+                      }
+                      label="Pull request re-opened"
+                    />
+                    <FormControlLabel
+                      className={classes.formLabelSpacing}
+                      control={
+                        <Checkbox
+                          checked={events.has('push')}
+                          onChange={this.handleEventsSelection}
+                          value="push"
+                        />
+                      }
+                      label="Push"
+                    />
+
+                    <FormControlLabel
+                      className={classes.formLabelSpacing}
+                      control={
+                        <Checkbox
+                          checked={events.has('release')}
+                          onChange={this.handleEventsSelection}
+                          value="release"
+                        />
+                      }
+                      label="Release or tag created"
+                    />
+                  </FormGroup>
+                </div>
               </FormControl>
             </ListItem>
-            <ListItem>
+            <ListItem className={classes.generalSpacing}>
               <TextField
                 id="select-access"
                 select
```

Comment:
Do I also make the "Access", "Project Language" and "Commands" list items flex and show them in just one row? @helfi92